src-tauri/src/telemetry_manager.rs

```diff
@@ -658,6 +669,7 @@ async fn get_telemetry_data_inner(
         cpu_utilization,
         cpu_make,
         gpu_make,
+        gpu_hash_rate_c29,
```

Comment:
We need to add it to the `extra_data` field. For some reason they can't expand the schema at the root level 🤷🏻‍♂️